examples/docs_snippets/docs_snippets/integrations/embedded_elt/sling_dagster_translator.py

```diff
@@ -11,11 +10,14 @@
 sling_resource = SlingResource(connections=[...])  # Add connections here
 
 
-@sling_assets(replication_config=replication_config)
+@sling_assets(
+    replication_config=replication_config,
+    dagster_sling_translator=DagsterSlingTranslator(),
+)
 def my_assets(context, sling: SlingResource):
     yield from sling.replicate(
+        context=context,
         replication_config=replication_config,
```

Comment:
@PedramNavid , while updating the examples I realized that `replication_config` also requires two definitions. I can take a stab at applying the same treatment here.

I also noticed that in all of the examples we showed passing the translator in the `replicate` call. Hopefully moving it to the decorator isn't too burdensome for the users.

Reply:
I've done the same thing with `replication_config` in this commit: https://github.com/dagster-io/dagster/pull/20564/commits/26f4e147e55a183bd86b0a39b8d7016940c4a6e9